### DISCUSSION TITLE
More Locations

### DISCUSSION THREAD

**Discussion by TheSuggestionGuy** [Stellarium/stellarium#3542] [Ideas]
Making locations is annoying especially when they have their own time rules.
Cities I want in the location list:
Taiohae, Polynesia
Easter Island, Chile
Puvirnituq, Northern America
Kangirsuk, Northern America
Kuujjuaq, Northern America
Nain, Northern America
Makkovik, Northern America
Clyde River, Northern America
Nome, Northern America
Alert, Northern America

**Comment by TheSuggestionGuy** [comment]
You (stellarium) can add more.

**Reply by gzotti** [reply]
Adding locations is trivial. Add your own.

**Reply by TheSuggestionGuy** [reply]
Searching: how to make your own location with their own set of time rules about daylight savings

**Reply by gzotti** [reply]
You assign the right time zones. Not "UTC-6" but "America/YourTown". If DST rules are different from the existing, there is no way. And we will not invent new DST rules here.

**Reply by TheSuggestionGuy** [reply]
Kk, but why can't we be able to manage when daylight savings starts and when daylight savings ends, or am I not getting it?

**Reply by gzotti** [reply]
Stellarium is using system libraries to retrieve this information from these well-known zone names. There is no way to define others (unless you start developing that).

**Reply by Vendrel** [reply]
I added my yard. Made the panoramic photo in Google Streetview app (you only have it if you had it back when it was available), aligned its orientation (North to North) in Photoshop, and now I can select it anytime. ChatGPT helped me out with the editing of landscape.ini file.
